modules/ref-example-kueue-resource-configurations.adoc

```diff
@@ -0,0 +1,211 @@
+
+:_module-type: REFERENCE
+
+[id='example-kueue-resource-configurations_{context}']
+= Example Kueue resource configurations
+
+[role='_abstract']
+These examples show how to configure Kueue resource flavors and cluster queues. 
+
+ifndef::upstream[]
+ifdef::self-managed[]
+[NOTE]
+====
+In {productname-short} {vernum}, {org-name} does not support shared cohorts.
+====
+endif::[]
+ifdef::cloud-service[]
+[NOTE]
+====
+In {productname-short}, {org-name} does not support shared cohorts.
+====
+endif::[]
+endif::[]
+
+
+//== NVIDIA GPUs without shared cohort
+// When AMD GPUs are supported, uncomment the above line and delete the following line
+== NVIDIA GPUs 
+
+=== NVIDIA RTX A400 GPU resource flavor
+
+[source,bash]
+----
+apiVersion: kueue.x-k8s.io/v1beta1
+kind: ResourceFlavor
+metadata:
+  name: "A400-node"
+spec:
+  nodeLabels:
+    instance-type: nvidia-node
+  tolerations:
+  - key: "HasGPU"
+    operator: "Exists"
+    effect: "NoSchedule"
+
+----
+
+=== NVIDIA RTX A1000 GPU resource flavor
+
+[source,bash]
+----
+apiVersion: kueue.x-k8s.io/v1beta1
+kind: ResourceFlavor
+metadata:
+  name: "A1000-node"
+spec:
+  nodeLabels:
+    instance-type: nvidia-node
+  tolerations:
+  - key: "HasGPU"
+    operator: "Exists"
+    effect: "NoSchedule"
+----
+
+=== NVIDIA RTX A400 GPU cluster queue
+
+[source,bash]
+----
+apiVersion: kueue.x-k8s.io/v1beta1
+kind: ClusterQueue
+metadata:
+  name: "A400-queue"
+spec:
+  namespaceSelector: {} # match all.
+  resourceGroups:
+  - coveredResources: ["cpu", "memory", "nvidia.com/gpu"]
+    - name: "A400-node"
+      resources:
+      - name: "cpu"
+        nominalQuota: 16
+      - name: "memory"
+        nominalQuota: 64Gi
+      - name: "nvidia.com/gpu"
+        nominalQuota: 2
+
+----
+
+=== NVIDIA RTX A1000 GPU cluster queue
+
+[source,bash]
+----
+apiVersion: kueue.x-k8s.io/v1beta1
+kind: ClusterQueue
+metadata:
+  name: "A1000-queue"
+spec:
+  namespaceSelector: {} # match all.
+  resourceGroups:
+  - coveredResources: ["cpu", "memory", "nvidia.com/gpu"]
+    flavors:
+    - name: "A1000-node"
+      resources:
+      - name: "cpu"
+        nominalQuota: 16
+      - name: "memory"
+        nominalQuota: 64Gi
+      - name: "nvidia.com/gpu"
+        nominalQuota: 2
+
+----
```

Comment:
Will this example be available in the docs?

Reply:
Would you prefer to include or omit it?

Reply:
Confirmed in DM to omit for now.